src/components/notification/NotificationNotice.vue

```diff
@@ -31,12 +32,16 @@ export default {
                 this.$emit('close')
 
                 // Timeout for the animation complete before destroying
-                setTimeout(() => {
+                this.timeOutID = setTimeout(() => {
                     this.$destroy()
                     removeElement(this.$el)
                 }, 150)
             }
         }
+    },
+
+    beforeDestroy() {
+        clearTimeout(this.timeOutID)
```

Comment:
@Arooba-git This should be unnecessary. Because `NotificationNotice` is intended to be programmatically opened and it won't be destroyed until the timeout expires. Please see the line 36.

Reply:
@Arooba-git I have reverted the changes in your fork.